statefun-examples/statefun-ridesharing-example/statefun-ridesharing-example-simulator/src/main/resources/META-INF/licenses/LICENSE.javax.annotation-api

```diff
@@ -360,400 +360,3 @@ litigation relating to this License shall be subject to the jurisdiction
 of the Federal Courts of the Northern District of California and the
 state courts of the State of California, with venue lying in Santa Clara
 County, California.
-
```

Comment:
I'm not an expert, but here's my opinion:

I don't feel comfortable editing a license we are forwarding:
The CDDL states 
```
Except as provided in Section 4.3, no one other than the license steward has the right to modify this License.
```
I'm not sure if "this License" means only the CDDL, or the LICENSE file of javax annotation api.

Why can't we forward the javax annotation api LICENSE file (unmodified, as requested by the license), but state in our NOTICE file, that we are using the javax annotation api under the terms of the CDDL?

I believe this approach is less risky (remember `Oracle is the initial license steward`)

Reply:
Hmmm, interesting point. My point came from other dual-licensed projects (like Jackson was in the past) that included two license files (ASLv2, LGPL) and you forwarded the one you selected. I was thinking to mirror that here, but I don't know if the assumption here is that you have to forward the information about being dual licensed.

Reply:
I think I agree with @rmetzger here.
Modying the LICENSE seems risky.
Relying on our NOTICE to clearly state which license we're using the dependency under the terms of, should be sufficient.